<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsMud</class>
 <widget class="QWidget" name="PrefsMud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="_hyperlinks">
     <property name="title">
      <string>MXP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="UseMxp_label">
          <property name="text">
           <string>Use MXP:</string>
          </property>
          <property name="buddy">
           <cstring>UseMxp</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="UseMxp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>On command</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>On query</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="IgnoreMxpColourChanges">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ignore MXP colour effects</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_custom_color">
        <item>
         <widget class="QCheckBox" name="UseCustomLinkColour">
          <property name="text">
           <string>Use custom link colour:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPickerButton" name="HyperlinkColour">
          <property name="baseSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="MudCanChangeLinkColour">
        <property name="text">
         <string>MUD can change link colour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UnderlineHyperlinks">
        <property name="text">
         <string>Underline hyperlinks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HyperlinkAddsToCommandHistory">
        <property name="text">
         <string>Save hyperlink mouse clicks in history</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="EchoHyperlinkInOutputWindow">
        <property name="text">
         <string>Echo hyperlink mouse clicks in output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_telnet">
     <property name="title">
      <string>Telnet</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QLabel" name="TerminalIdentification_label">
          <property name="text">
           <string>Terminal identification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TerminalIdentification"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="DisableCompression">
        <property name="text">
         <string>Disable compression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Naws">
        <property name="text">
         <string>Negotiate about window size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CarriageReturnClearsLine">
        <property name="text">
         <string>Carriage-return clears line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Utf8">
        <property name="text">
         <string>UTF-&amp;8 (Unicode)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ConvertGaToNewline">
        <property name="text">
         <string>Convert IAC EOR/GA to newline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="NoEchoOff">
        <property name="text">
         <string>Ignore &quot;Echo Off&quot; server commands</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_command_stacking">
     <property name="title">
      <string>Command Stacking</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="EnableCommandStack">
        <property name="text">
         <string>Enable &amp;command stacking with prefix:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CommandStackCharacter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>600</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
